counting_prawns.10s.js

```diff
@@ -86,16 +88,18 @@ const getPrDividedByReposAndAssignedToYou = () => (
     prDividedByRepos.map(prs =>
       prs.filter(pr => filterPrsByLoginNames(pr))
     )
-  )
+  ).catch((message) => message)
 );
 
-const getTotalPrAssignedToYou = (prDividedByReposAssignedToYou) => (
-  prDividedByReposAssignedToYou.reduce((acc, prDividedByRepos) => (
-    acc + prDividedByRepos.length
-  ), 0)
-);
+const getTotalPrAssignedToYou = (prDividedByReposAssignedToYou) => {
+  if(prDividedByReposAssignedToYou instanceof Array) {
+    return prDividedByReposAssignedToYou.reduce((acc, prDividedByRepos) => (
+      acc + prDividedByRepos.length
+    ), 0)
+  } else return prDividedByReposAssignedToYou;
+};
 
-const getEmoji = num => (num > 0 ? ':eyeglasses:' : ':palm_tree:');
+const getEmoji = num => (num > 0 ? ':eyeglasses:' : num === 0? ':palm_tree:' : ':x:');
```

Comment:
nested ternary are just not understandable. Refactor this, into a documented function.